hail_scripts/reference_data/config.py

```diff
@@ -202,8 +203,25 @@ def custom_gnomad_select_v3(ht):
     'screen': {
         '38': {
             'path' : 'gs://seqr-reference-data/GRCh38/ccREs/GRCh38-ccREs.ht',
-            'select': {'region_type': 'target'}
-        }
+            'enum_selects': [{
+                'src': 'target',
+                'dst': 'regionType_ids',
+                'mapping': hl.dict(
```

Comment:
A reasonable optimization here is to store just the list (without the `hl.dict` and `hl.enumerate`) so that we can pass just a list to `seqr` on the globals.  the `hl.dict` and `hl.enumerate` would be called at mapping time.